### PR TITLE
Support initializer not in inputs

### PR DESCRIPTION
**Pull request by scxiao**

### CODE REVIEW

File: src/onnx/onnx.cpp [hunk 1]
```diff
@@ -1426,17 +1426,14 @@ struct onnx_parser
         for(auto&& f : graph.initializer())
         {
             initializer_data[f.name()] = f;
```
Comment (pfultz2) [inline src/onnx/onnx.cpp]: Do we even need the `initializer_data` map now? We can just create the literals instead.

File: src/onnx/onnx.cpp [hunk 2]
```diff
@@ -1426,17 +1426,14 @@ struct onnx_parser
...
             const std::string& name = input.name();
-            // Does the input have an initializer?
-            if(contains(initializer_data, name))
-            {
-                auto t             = initializer_data[name];
-                instructions[name] = prog.add_literal(parse_tensor(t));
-            }
-            else
+            // input not in initializer_data, so it is a real input
+            if(!contains(initializer_data, name))
```
Comment (pfultz2) [inline src/onnx/onnx.cpp]: This could be `!contains(instructions, name)` instead.

Comment (scxiao) [reply src/onnx/onnx.cpp]: changed accordingly

File: src/onnx/onnx.cpp (hunk 1 above)
Comment (scxiao) [reply src/onnx/onnx.cpp]: removed it.

**Pull request merged**